README.rst

```diff
@@ -29,10 +29,10 @@ Cite this software
 
 The references to be acknowledged are ::
 
-    Gramfort et al. OpenMEEG: opensource software for quasistatic
+    Gramfort A, Papadopoulo T, Olivi E, Clerc M. OpenMEEG: opensource software for quasistatic
     bioelectromagnetics. Biomedical engineering online (2010) vol. 9 (1) pp. 45
 
-    Kybic et al. Generalized head models for MEG/EEG: boundary element method
+    Kybic J, Clerc M, Abboud T, Faugeras O, Keriven R, Papadopoulo T. Generalized head models for MEG/EEG: boundary element method
```

Comment:
@massich why is this here?

Reply:
'cos I messed up triggering travis. It should not be here. since it is already in master. So I don't really understand it.